components/UiValidationInput.vue

```diff
@@ -57,11 +78,21 @@ export default {
     isNeedToCheck() {
       return this.validateOn
     },
+    isValidCarrierUbn() {
+      if (this.validateField !== 'carrierUbn') return true
+      const reg = /^\d{8}$/
+      return reg.test(this.value)
+    },
+    disable() {
+      return this.carrierType === '請選擇'
+    },
   },
   methods: {
     changeHandler(e) {
+      this.hasChange = true
       this.$v.value.$touch()
       this.$emit('input', e.target.value)
+      this.value = e.target.value
```

Comment:
`this.value` 是 props，這邊應避免直接 mutate props，devtools console 應也有提示：
```
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "value"

found in

---> <UiValidationInput> at components/UiValidationInput.vue
       <SubscribeFormReceipt> at components/SubscribeFormReceipt.vue
         <SubscribeForm> at components/SubscribeForm.vue
           <Pages/papermag/Plan/index.vue> at pages/papermag/_plan/index.vue
             <Nuxt>
               <Layouts/default.vue> at layouts/default.vue
                 <Root>
```

Reply:
我現在看忘了當初為什麼寫這行了 ><
（應該有 94 行就夠）
感謝提醒！